### ISSUE TITLE
Questions about the pretrain model

### ISSUE THREAD

**Issue by zhouzhou9999** [thuml/Universal-Domain-Adaptation#14]
hello！Can you share the pre-trained resnet-50 model and officehome label files?
Very urgent and thanks!

**Comment by zhouzhou9999** [comment]
I have downloaded the resnet-50 model on the Internet（https://download.pytorch.org/models/resnet50-19c8e357.pth）, but it is not very feasible。

**Comment by zhouzhou9999** [comment]
我自己生成了officehome标签，但是不知道标签文档标记数字的不同，会影响源域共有类得标签，然后会影响结果？

**Comment by Dayro-9** [comment]
> hello！Can you share the pre-trained resnet-50 model and officehome label files?
> Very urgent and thanks!
你好，请问关于预训练模型的问题，你是怎么解决的？
